packages/one-app-bundler/README.md

```diff
@@ -164,6 +164,35 @@ under `bundler.purgecss.paths`. The example below illustrates how we would add
 }
 ```
 
+Additional `purgecss` options. Please refer to the [`purgecss Options Documentation`](https://github.com/FullHuman/purgecss)
+before enabling any of the following:
+
+```json
+{
+  "one-amex": {
+    "bundler": {
+      "purgecss": {
+      "paths": ["node_modules/some-lib/src/**/*.{js,jsx}"],
+      "extractors": [{
+        "extractor": "purgeJs",
+        "extensions": [
+          "js"
+        ]
+      }],
+      "fontFace": false,
+      "keyframes": false,
+      "variables": false,
+      "whitelist": ["random",
+        "yep",
+        "button"],
+      "whitelistPatterns": ["red"],
+      "whitelistPatternsChildren": ["blue"]
```

Comment:
```suggestion
        "paths": ["node_modules/some-lib/src/**/*.{js,jsx}"],
        "extractors": [{
          "extractor": "purgeJs",
          "extensions": [
            "js"
          ]
        }],
        "fontFace": false,
        "keyframes": false,
        "variables": false,
        "whitelist": [
          "random",
          "yep",
          "button"
        ],
        "whitelistPatterns": ["red"],
        "whitelistPatternsChildren": ["blue"]
```

Reply:
I don't really know what went on with the indentation there 🤨